mitiq/tests/pyquil_utils.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+from pyquil import Program
+# Backend and Noise simulation
+from pyquil import get_qc
+from pyquil.noise import append_kraus_to_gate
+from pyquil.gates import X, Y, Z, MEASURE
+
+QVM = get_qc('1q-qvm')
+
+# Set the random seeds for testing
+QVM.qam.random_seed = 1337
```

Comment:
@andreamari FYI this is how to add a random seed to the QVM. It's more hidden than it should be to be honest...